addon/components/sl-button.js

```diff
@@ -145,45 +156,57 @@ export default Ember.Component.extend( TooltipEnabled, {
         if ( label ) {
             return label;
         }
-    }.property( 'label', 'pending', 'pendingLabel' ),
+    }),
 
     /**
      * Converted size string to Bootstrap button class
      *
-     * @function sizeClass
+     * @function
      * @observes size
```

Comment:
@observes can be removed due to the intended use of https://github.com/notmessenger/jsdoc-plugins#emberobservestag